pkg/codegen/dotnet/templates.go

```diff
@@ -93,6 +93,12 @@ namespace {{.Namespace}}
             using var stream = assembly.GetManifestResourceStream("{{.Namespace}}.version.txt");
             using var reader = new StreamReader(stream ?? throw new NotSupportedException("Missing embedded version.txt file"));
             version = reader.ReadToEnd().Trim();
+            var parts = version.Split("\n");
+            if (parts.Length == 2)
+            {
+                // The first part is the provider name.
+                version = parts[1];
```

Comment:
While not strictly necessary, given my other comment:

```suggestion
                version = parts[1].Trim();
```